xml/selinux.xml

```diff
@@ -313,99 +291,45 @@ system_u:object_r:var_t var</screen>
    recommended to use a modular policy and not a monolithic policy. Modular
    policies are much easier to manage.
   </para>
- </sect1> -->
+ </sect1>
  
  <sect1 xml:id="sec-selinux-install">
-  <title>Installing &selnx; packages and modifying &grub;</title>
-
+  <title>Installing &selnx; packages</title>
   <para>
-   The easiest way to make sure that all &selnx; components are installed is
-   by using &yast;. The procedure described below shows what to do on an
-   installed &productname;:
+    If you are installing &selnx; with Zypper, search for all of the relevant
+    packages with the <command>--search-descriptions</command> option. You  
+    will see a list like the following example:
+  </para>
+  <screen>&prompt.user;<command>zypper se --search-descriptions selinux</command>
+libselinux-devel 
+libselinux1 
+libselinux1-32bit
+libsemanage-devel
+libsemanage1 
+libsepol-devel 
+libsepol1 
+mcstrans 
+policycoreutils 
+python3-policycoreutils
+python3-selinux 
+python3-semanage
+python3-setools 
+restorecond
+selinux-tools
+setools-console
+setools-devel 
+setools-java
+setools-libs
+setools-tcl
+  </screen>
+  <para>
+    Install of them to ensure complete functionality.
```

Comment:
```suggestion
    Install all of them to ensure complete functionality.
```

Reply:
remote this paragraph and change it to:
zypper in restorecond policycoreutils setools-console
this pulls in everything most users need

Reply:
:+1: